base/applications/mspaint/winproc.cpp

```diff
@@ -384,6 +385,7 @@ void CMainWindow::ProcessFileMenu(HMENU hPopupMenu)
     EnableMenuItem(hPopupMenu, IDM_FILEASWALLPAPERPLANE,     ENABLED_IF(g_isAFile && isBMP));
     EnableMenuItem(hPopupMenu, IDM_FILEASWALLPAPERCENTERED,  ENABLED_IF(g_isAFile && isBMP));
     EnableMenuItem(hPopupMenu, IDM_FILEASWALLPAPERSTRETCHED, ENABLED_IF(g_isAFile && isBMP));
+    EnableMenuItem(hPopupMenu, IDM_FILESEND,                 ENABLED_IF(PathFileExists(g_szFileName)));
```

Comment:
Wouldn't this enable the menu only if the file has been already saved once?

Reply:
Fixed.